config/initializers/fast_gettext.rb

```diff
@@ -0,0 +1,14 @@
+# db access is cached <-> only first lookup hits the db
+require "fast_gettext/translation_repository/db"
+FastGettext::TranslationRepository::Db.require_models #load and include default models
+
+FastGettext.add_text_domain(
+  'hmis_warehouse', 
```

Comment:
Is this the database name?  If it is, it needs to come from the database.yml file.